app/tasks/launch.js

```diff
@@ -105,6 +114,46 @@ let self = {
     return self.launch(exe_path, [], opts)
   },
 
+  launch_html_cave: async function(opts, cave) {
+    let game = await db.find_one({_table: 'games', id: cave.game_id})
+    let app_path = CaveStore.app_path(cave.install_location, opts.id)
+    if (!fs.existsSync(path.join(app_path, 'index.html'))) {
+      throw new Error('html game missing index.html')
+    }
+    let win = new BrowserWindow({
+      title: game.title,
+      icon: './static/images/itchio-tray-x4.png',
+      width: 1280, height: 720,
```

Comment:
games can specify actual width/height but we don't have that info yet, just opened https://github.com/leafo/itch.io/issues/310